docker-compose.yml

```diff
@@ -1,53 +1,83 @@
 version: '2'
 services:
 
-  postgres:
-    image: postgres
+  database:
+    image: camptocamp/postgis:9.5
```

Comment:
as long as we are going to create two different databases as above which are then configured in the local_settings.py. This should be valid only for production environment while in dev we use the django default db.

In theory if we'd like better segregation of duty and microservices' concept then we should run two distinct containers one from a plain postgres image and another from a postgis one with attached a data container volume for each in case of easier backup and data migration policies. Maybe in future this makes sense.